### PR TITLE
add information about Endpoint Manager

### PR DESCRIPTION
**Pull request by VLG17**
https://github.com/MicrosoftDocs/windows-itpro-docs/issues/8906

### CODE REVIEW

File: windows/security/threat-protection/microsoft-defender-atp/enable-exploit-protection.md [hunk 1]
```diff
@@ -125,6 +125,21 @@ The result will be that DEP will be enabled for *test.exe*. DEP will not be enab
...
 
+## Microsoft Endpoint Manager
+
+1. In Microsoft Endpoint Manager, click **Endpoint Security** > **Attack surface reduction** 
+
+2. Click **Create Policy**, select **Platform** and under **Profile** choose **Exploit Protection**. Click **Create**.
+
+3. Enter a name and a description and click **Next**.
+
+4. Click **Select XML File** and browse to the location of the exploit protection XML file, select it, and click **Next**.
```
Comment (illfated) [inline windows/security/threat-protection/microsoft-defender-atp/enable-exploit-protection.md:136]: 1: add missing sentence ending period dot, 2 + 3: add Oxford comma (sentence logic), 4: readability adjustment (debatable)
```suggestion
1. In Microsoft Endpoint Manager, click **Endpoint Security** > **Attack surface reduction**. 

2. Click **Create Policy**, select **Platform**, and under **Profile** choose **Exploit Protection**. Click **Create**.

3. Enter a name and a description, and click **Next**.

4. Click **Select XML File** and browse to the location of the exploit protection XML file, then select it and click **Next**.
```

**Review by illfated — APPROVED** [review]
Looks good to me, even though the copy review might reveal whether I missed something.

**Review by denisebmsft — APPROVED** [review]
Thank you @VLG17 @illfated

**Comment by illfated** [comment]
Thank you for the additional edits. 👍🏻

**Pull request merged**